pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -83,7 +83,13 @@ func (podDisruptionBudgetStrategy) AllowCreateOnUpdate() bool {
 
 // ValidateUpdate is the default update validation for an end user.
 func (podDisruptionBudgetStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	return validation.ValidatePodDisruptionBudget(obj.(*policy.PodDisruptionBudget))
+	newPDB := obj.(*policy.PodDisruptionBudget)
+	oldPDB := old.(*policy.PodDisruptionBudget)
+	// Apply tighter validation on update only if the existing object also passes the tighter validation
+	if allErrs := validation.ValidatePodDisruptionBudgetStrict(oldPDB); len(allErrs) == 0 {
+		return allErrs
```

Comment:
this skips validation entirely if the old PDB passed strict validation... that isn't what we want.